recipe/patches/0001-Fix-software-version-test.patch

```diff
@@ -1,3 +1,5 @@
+https://github.com/conda-forge/pyorc-feedstock/pull/18#issuecomment-1012846145
+
```

Comment:
Ah would suggest adding this to `meta.yaml` as a comment. Also would be worth including the link to the commit

Reply:
Done 👍